common/test/src/ui/tst_MapDocument.cpp

```diff
@@ -464,34 +464,66 @@ TEST_CASE_METHOD(MapDocumentTest, "canUpdateLinkedGroups")
 
 TEST_CASE_METHOD(MapDocumentTest, "newDocument")
 {
+  const auto fgdContent = R"(
```

Comment:
This version is fine, but you should use `TestEnvironment` for this. You can create files by calling `TestEnvironment::createFile`. The benefit is that it cleans up after itself.